templates/saphana/rhel8.saphana.yaml

```diff
@@ -0,0 +1,215 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  annotations:
+    defaults.template.kubevirt.io/disk: rootdisk
+    description: Template for Red Hat Enterprise Linux 8.4 for SAP HANA workloads. Please consult the SAP HANA guide for node setup requirements.
+    iconClass: icon-rhel
+    name.os.template.kubevirt.io/rhel8.4: Red Hat Enterprise Linux 8.0 or higher
+    openshift.io/display-name: Red Hat Enterprise Linux 8.4 VM for SAP HANA workloads
+    openshift.io/documentation-url: https://github.com/kubevirt/common-templates
+    openshift.io/provider-display-name: Red Hat - Tech Preview
+    openshift.io/support-url: https://github.com/kubevirt/common-templates/issues
+    operator-sdk/primary-resource: openshift-cnv/ssp-kubevirt-hyperconverged
+    operator-sdk/primary-resource-type: SSP.ssp.kubevirt.io
+    tags: hidden,kubevirt,virtualmachine,linux,rhel,sap,hana
+    template.kubevirt.io/provider: Red Hat - Tech Preview
+    template.kubevirt.io/provider-support-level: Experimental
+    template.kubevirt.io/provider-url: https://www.redhat.com
+    template.kubevirt.io/version: v1alpha1
+    template.openshift.io/bindable: "false"
+  labels:
+    app.kubernetes.io/component: templating
+    app.kubernetes.io/managed-by: ssp-operator
+    app.kubernetes.io/name: common-templates
+    app.kubernetes.io/part-of: hyperconverged-cluster
+    app.kubernetes.io/version: 4.10.1
+    flavor.template.kubevirt.io/tiny: "true"
+    os.template.kubevirt.io/rhel8.4: "true"
+    template.kubevirt.io/type: base
+    template.kubevirt.io/version: v0.19.3
```

Comment:
@yaacov The UI should keep working for the template. Van we drop some of the labels, without confusing the UI?

Reply:
Yes, UI should not crash by dropping the labels, we have fallback in case we can't find some labels we use in the UI.

we use
openshift.io/support-url - The UI use it to show a link to support site

AFAIK we do not use
operator-sdk/primary-resource
operator-sdk/primary-resource-type

the versions are used in case we have 2 templates with the same os/workload/size we show the last one, if we can't find the version we will show all (all in 4.11 or the first one in 4.10)

Reply:
Thanks,
I modified the annotations
```
openshift.io/documentation-url
openshift.io/support-url
```

removed
```
operator-sdk/primary-resource:
operator-sdk/primary-resource-type:
template.kubevirt.io/version:
```
removed lables
```
app.kubernetes.io/managed-by: ssp-operator
app.kubernetes.io/name: common-templates
app.kubernetes.io/part-of: hyperconverged-cluster
app.kubernetes.io/version: 4.10.1
template.kubevirt.io/version: v0.19.3
vm.kubevirt.io/template.revision: "1"
vm.kubevirt.io/template.version: v0.19.3
```
and verified that it works on 4.10.10 UI